packages/elements/src/select/index.ts

```diff
@@ -192,11 +185,24 @@ export class Select extends ControlElement implements MultiValue {
   @property({ type: Boolean, reflect: true })
   public opened = false;
 
+  private _error = false;
+
   /**
   * Set state to error
+  * @param value error value
   */
   @property({ type: Boolean, reflect: true })
-  public error = false;
+  public set error (value: boolean) {
+    const oldValue = this._error;
+    if (oldValue !== value) {
+      this._error = value;
+      this.ariaInvalid = String(value);
```

Comment:
It would set `aria-invalid="null"` when I set `error` as `null`, The some screen reader e.g. NVDA also read as invalid.